core/foundation/src/TClassEdit.cxx

```diff
@@ -1274,15 +1274,13 @@ int TClassEdit::GetSplit(const char *type, vector<string>& output, int &nestedLo
 
 string TClassEdit::CleanType(const char *typeDesc, int mode, const char **tail)
 {
-   static const char* remove[] = {"class", "const", "volatile", nullptr};
-   auto initLengthsVector = []() {
-      std::vector<size_t> create_lengths;
-      for (int k=0; remove[k]; ++k) {
-         create_lengths.push_back(strlen(remove[k]));
-      }
-      return create_lengths;
-   };
-   static std::vector<size_t> lengths{ initLengthsVector() };
+   constexpr static std::array<const char *, 3> remove{"class", "const", "volatile"};
+   constexpr static auto lengths = []() constexpr {
+      std::array<std::size_t, 3> ret{};
```

Comment:
```suggestion
      std::array<std::size_t, std::size(remove)> ret{};
```
To avoid the magic number three, if one wants, but yeah not a big deal! Super nice @vepadulano, I didn't know that for loops can be `constexpr` :+1::+1: